.openshift-ci/post_tests.py

```diff
@@ -127,6 +139,7 @@ def run(self, test_outputs=None):
         self.collect_service_logs()
         if self._check_stackrox_logs:
             self.check_stackrox_logs()
+        self.add_test_results(test_results)
         self.store_artifacts(test_outputs)
```

Comment:
To keep the order consistent with `StoreArtifacts.run`:
```suggestion
        self.store_artifacts(test_outputs)
        self.add_test_results(test_results)
```